src/Core/Resources/SharedResources.en.resx

```diff
@@ -665,12 +665,12 @@
     <value>Automatic Enrollment</value>
   </data>
   <data name="ResetPasswordAutoEnrollDescription" xml:space="preserve">
-    <value>All users will be automatically enrolled in password reset once their invite is accepted.</value>
+    <value>All users will be automatically enrolled in password reset once their invite is accepted and will not be allowed to withdraw.</value>
   </data>
   <data name="ResetPasswordAutoEnrollWarning" xml:space="preserve">
     <value>Users already in the organization will not be retroactively enrolled in password reset. They will need to self-enroll before administrators can reset their master password.</value>
   </data>
   <data name="ResetPasswordAutoEnrollCheckbox" xml:space="preserve">
-    <value>Automatically enroll new users</value>
+    <value>Require new users to be enrolled automatically</value>
```

Comment:
Does this need a period?
```suggestion
    <value>Require new users to be enrolled automatically.</value>
```

Reply:
I do not believe so. Other checkbox descriptions do not end with a period